ansible/docker_site.yml

```diff
@@ -58,6 +58,7 @@
                                                       "python_code_coverage", "cpp_code_coverage"]}
     - {role: ci/code_style_check/python, when: run_build, tags: ["python_code_style_check", "code_style_check"]}
     - {role: ci/code_style_check/cpp, when: run_build, tags: ["cpp_code_style_check", "code_style_check"]}
+    - {role: ci/code_style_check/license, when: run_build, tags: ["check_license", "code_style_check"]}
```

Comment:
please remove "code_style_check" since it will fail in all existing projects :)

Reply:
done in c1baab4